bin/file.js

```diff
@@ -32,7 +32,12 @@ module.exports = function (program, file) {
   var transformer = new Transformer({transformers: transformers});
   transformer.readFile(file[0]);
   transformer.applyTransformations();
-  transformer.writeFile(program.outFile);
+  if(program.stdout){
```

Comment:
Can we change this to `if(program.outFile === '-'){`, as is standard convention for stdout in Unix?